server/src/main/scala/io/delta/standalone/internal/JsonPredicates.scala

```diff
@@ -0,0 +1,417 @@
+/*
+ * Copyright (2021) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.delta.standalone.internal
+
+// NOTE: This file should be kept in sync with the corresponding file
+// in the spark connector.
+
+import scala.collection.mutable.ListBuffer
+
+import com.fasterxml.jackson.annotation.{JsonIgnore, JsonSubTypes, JsonTypeInfo}
+
+/**
+ * The evaluation context in which operations will be evaluated.
+ *
+ * @param partitionValues The partition values associated with parquet files.
+ */
+case class EvalContext(partitionValues: Map[String, String])
+
+/**
+ * The data types supported by the filtering operations.
+ */
+object OpDataTypes {
```

Comment:
Seems like there's no way to use the same Jsonpredicates in both server/ and spark/?

Reply:
Ya, I had checked with Ryan Zu as well.
The way would be to make a common libraries, but that will trigger additional dependencies.
We can do that as a follow up change.